scripts/main_app.py

```diff
@@ -0,0 +1,15 @@
+import streamlit as st
+from pages import demo_playbooks, playbook_builder
+from streamlit import session_state as ss
+
+if "current_page" not in st.session_state:
+    st.session_state["current_page"] = "demo_playbooks"
+
+if "playbook_choosen" not in st.session_state:
```

Comment:
@pavangudiwada typo in `choosen` - should be `chosen``

Reply:
Fixed